cr-examples/eventstreams-georeplicator/07-georeplicator.yaml

```
@@ -19,5 +19,5 @@ metadata:
     # The value of this label needs to match the metadata.name of your EventStreams cluster
     eventstreams.ibm.com/cluster: minimal-prod
 spec:
-  version: 11.0.4
+  version: 11.1.0
```

And the same for the ESGR CR
```suggestion
  version: latest
```